### ISSUE TITLE
Getting older VRM character to look at the camera 

### ISSUE THREAD

**Issue by alankent** [pixiv/three-vrm#1173]
I found some hints in the migration guide, but I was wondering if I could get some advice. I have some pre-1.0 VRM files that I loaded into a web browser. I got the blendshapes working (yay!). I then tried to get the character to look at the camera. The default seemed to be the BACK of the head would look towards the camera. So I flipped the order of subtraction, but that did not work either (always looked too high up in the sky). A snippet of the final code that works for me (that does not look right) is:

```
function animate() {
  requestAnimationFrame(animate);
  if (currentVrm) {
    // Look at camera
    if (currentVrm.humanoid) {
      const headBone = currentVrm.humanoid.getNormalizedBoneNode('head')
      if (headBone) {
        const lookAt = new THREE.Vector3();
        lookAt.subVectors(headBone.position, camera.position)
        lookAt.setY(lookAt.y + 3)
        headBone.lookAt(lookAt);
      }
    }
    renderer.render( scene, camera );
  }
  animate();
}
```

I had to add 3 to the y value to stop it lookup up in the sky. If I understand some comments, it sounds like the Y got inverted? So maybe have to reverse the order of the Y subtraction now compared to X and Z? 

Advice appreciated! The next step is to stop the head turning beyond a certain angle (very weird seeing the head do a full 360 degree rotation!!!). And then get the eyes to look at the camera (e.g. get head turn 1/2 strength so the head turns half way to the camera, up to a limit, while the eyes turn fully to the camera

**Comment by alankent** [comment]
Oh, while I could not find up to date documentation, I did just find the examples like lookat-advanced.html. I got eye tracking going and it has the head turn with limits, so I think I should be fine once I have a few minutes to pull that code apart.

**Comment by 0b5vr** [comment]
If you want to make the VRM look at the camera, you can just simply say `vrm.lookAt.target = camera` .

**Comment by alankent** [comment]
Thanks for the reply. That makes the eyes turn towards the camera, but I was trying to get the head to turn as well. I found the advanced example, and got it to limit how far the eyes would turn (by limiting pitch and yaw). (I am not sure the rate of change code there is quite right, but eyes should dart quickly to the new target anyway so I removed it.) I assume I will need to do something similar with the head turn (with a limit on the turn speed of the head for realism). I want to get the head to turn  half of the eye turn angle. It may be I can just use the pitch and yaw from the eye turn, halve it, then use that for the target head rotation. I will see how it goes!
